src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
+dotnet build System.Device.Gpio.sln
+```
+This builds the main System.Device.Gpio assembly together with its test assembly. Before running the tests, you need to:
+- Connect an LED in series with a matching resistor to BCM Pin 18 (physical Pin 12) 
+- Connect BCM Pins 12 and 16 (physical Pins 32 and 36) with a cable
```

Comment:
We also connect a resistor between the two pins to ensure no short circuiting and protect your board.